docker/base-images/base-admin-tools.Dockerfile

```diff
@@ -8,9 +23,12 @@ RUN apk add --update --no-cache \
     jq \
     mysql-client \
     postgresql-client \
-    py-pip \
+    py3-pip \
     expat \
-    tini \
-    && pip install cqlsh
+    tini
+
+COPY --from=builder /opt/venv /opt/venv
+
+ENV PATH="/opt/venv/bin:$PATH"
```

Comment:
I'd run `cqlsh --version` in a step after building this container to validate the installation.